documentation/API_documentation/QoSProfile_Mapping_Table.md

```diff
@@ -1,11 +1,25 @@
 # QoS Profiles Mapping Table (REFERENCE DRAFT)
 
-| Profiles | User Info | 5Qi mapping |
+Here are two possible examples of mapping tables from QoS Profiles to provided connectivity characteristics. The second example describes a case, where the network is specially prepared in well define geographical areas for supporting Quality on Demand needs.
+
+Example 1: Considering wide-area network usage.
+| QOS Profile labels | Network Service Description | Network Service Constraints |
+|----------| --------- | ----------- |
+| QOS\_E   | Latency stays stable under congestion (throughput upto 2Mbps). The application bitrate should not exceed 2Mbps. | The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
+| QOS\_L   | DL upto 100Mbps (unlimited?). The 5G System throughput is prioritized up to 100Mbps. At high load, the throughput is capped at 100Mbps and can be degraded to lower effective throughput (without a minimum level).  | The service is only provided, when the UE is located within the home network. |
```

Comment:
```suggestion
| QOS\_L   | DL upto 100Mbps. The 5G System throughput is prioritized up to 100Mbps. At high load, the throughput is capped at 100Mbps and can be degraded to lower effective throughput (without a minimum level).  | The service is only provided, when the UE is located within the home network. |
```
Suggestion to delete "(unlimited?)"

"The service is only provided, when the UE is located within the home network." ... that is an operator specific decision, e.g. can be offered across multiple subsidiaries and/or also in partner networks.

Reply:
ok, that it a left over from the original table